### ISSUE TITLE
Can not install VelaUX on windows

### ISSUE THREAD

**Issue by liykfrank** [kubevela/kubevela#3920]
**Describe the bug**

I followed the guide https://kubevela.net/docs/install#4-install-velaux to install vela UI.
However the addon cannot be enabled due to some system requriement miss match.

**To Reproduce**

Follow this guide https://kubevela.net/docs/install#4-install-velaux :
1. install vela cli on windows
2. install kube vela core with the installed Cli.
3. Enable UI addon with `C:\vela\vela.exe  addon enable velaux --version v1.3.4` .
-Error: addon velaux system requirement miss match: vela cli/ux version: v1.4.0-alpha.3  require: >=v1.3.0-beta.2

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/26363251/168812730-d4075164-3f1a-4e05-abad-a07da4962a74.png)

**KubeVela Version**
PS D:\>  C:\vela\vela.exe  version
CLI Version: v1.4.0-alpha.3
Core Version: 1.4.0-alpha.3
GitRevision: git-f447e6c   
GolangVersion: go1.17.9

**Cluster information**
kubectl version
Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.6", GitCommit:"f59f5c2fda36e4036b49ec027e556a15456108f0", GitTreeState:"clean", BuildDate:"2022-01-19T17:33:06Z", GoVersion:"go1.16.12", Compiler:"gc", Platfor
m:"windows/amd64"

**Additional context**

**Comment by barnettZQG** [comment]
@liykfrank Please reinstall the vela core with the latest version 1.3.4.

```
curl -fsSl https://kubevela.io/script/install.sh | bash -s 1.3.4

vela --version v1.3.4

vela addon enable velaux version=v1.3.4
```

**Comment by liykfrank** [comment]
> @liykfrank Please reinstall the vela core with the latest version 1.3.4.
> 
> ```
> curl -fsSl https://kubevela.io/script/install.sh | bash -s 1.3.4
> 
> vela --version v1.3.4
> 
> vela addon enable velaux version=v1.3.4
> ```

@barnettZQG , thank you for your prompt response. I did a quick test it works well with 1.3.4.  
I guess it's something related with the latest version.
BTW, I tried to manually downgrade vela cli, it seems the artifact downloaded from the release notes page doesn't have a proper file extension.
I have to rename the cli from vela to vela.exe.
Why not do this in the build pipeline?
![image](https://user-images.githubusercontent.com/26363251/168822915-c14ea90f-aead-462a-a5f2-f237286342e9.png)
